projects/1.20/assets/rats/rats/lang/zh_cn.json

```
@@ -0,0 +1,1028 @@
+  "advancement.ratlantis.rat_upgrade_archeologist.title": "温新知故",
+  "advancement.ratlantis.rat_upgrade_buccaneer.desc": "合成老鼠升级：海盗，为老鼠提供了用不着的16世纪武装",
+  "advancement.ratlantis.rat_upgrade_buccaneer.title": "是的船长！",
+  "advancement.ratlantis.rat_upgrade_mount_biplane.desc": "合成老鼠升级：双翼机坐骑，然后获得你自己的王牌飞行员老鼠",
+  "advancement.ratlantis.rat_upgrade_mount_biplane.title": "霍鼠德·休斯",
+  "advancement.ratlantis.rat_upgrade_nonbeliever.desc": "合成老鼠升级：非信者，可以完爆一切的老鼠升级",
+  "advancement.ratlantis.rat_upgrade_nonbeliever.title": "对于非信者来说战神算什么？",
+  "advancement.ratlantis.rat_upgrade_psychic.desc": "合成老鼠升级：灵媒，并且拥有自己的通灵鼠",
+  "advancement.ratlantis.rat_upgrade_psychic.title": "才不是U盘",
+  "advancement.ratlantis.rat_upgrade_ratinator.desc": "合成老鼠升级：终结鼠，将致命的赛博老鼠变为现实",
+  "advancement.ratlantis.rat_upgrade_ratinator.title": "如同泪水消失在雨中",
+  "advancement.ratlantis.ratglove_petals.desc": "利用九个鼠套花合成鼠套花瓣",
+  "advancement.ratlantis.ratglove_petals.title": "花的力量",
+  "advancement.ratlantis.ratlantean_ratbot.desc": "遭遇鼠西族机器鼠，旧时代的遗物。",
+  "advancement.ratlantis.ratlantean_ratbot.title": "不锈钢老鼠",
+  "advancement.ratlantis.ratlantean_spirit.desc": "遇到湮没已久的鼠西族鬼魂",
+  "advancement.ratlantis.ratlantean_spirit.title": "一兆死灵遗灰",
+  "advancement.ratlantis.ratlantis.desc": "使用肥厚奶酪令牌在地上随处创建通往鼠西州的入口。并通过入口进入！",
+  "advancement.ratlantis.ratlantis.title": "鼠西洲！",
+  "advancement.ratlantis.ratlantis_armor.desc": "获得一整套的鼠西族战甲",
+  "advancement.ratlantis.ratlantis_armor.title": "贴金大佛！",
+  "advancement.ratlantis.root.desc": "一个失落的帝国",
+  "advancement.ratlantis.root.title": "鼠西洲",
+  "advancement.ratlantis.token.desc": "从老鼠身上获得大块奶酪令牌！做得好！",
+  "advancement.ratlantis.token.title": "抽卡！十连！全是SSR！！",
+  "advancement.ratlantis.upgrade_combiner.desc": "合成老鼠升级组合器，并开始将这些升级混合在一起",
+  "advancement.ratlantis.upgrade_combiner.title": "最  大  出  力！！！",
+  "advancement.ratlantis.vial_of_sentience.desc": "使用鼠西族魂火和野鼠爪制造知性小瓶。确保将其扔向野蛮鼠西族！",
+  "advancement.ratlantis.vial_of_sentience.title": "我冒的险是经过计算的……",
+  "advancement.rats.all_hats.desc": "收集所有帽子！",
+  "advancement.rats.all_hats.title": "帽子收藏家",
+  "advancement.rats.ball_of_filth.desc": "用大量的垃圾制作一个污渍球",
+  "advancement.rats.ball_of_filth.title": "洁净近乎圣洁！",
+  "advancement.rats.black_death.desc": "遭遇黑死神，是一名被雷击召唤的着魔瘟疫医生",
+  "advancement.rats.black_death.title": "群魔乱舞！",
+  "advancement.rats.cheese.desc": "将一块奶酪变成四个奶酪",
+  "advancement.rats.cheese.title": "我们忘拿饼干了！",
+  "advancement.rats.cheese_stick.desc": "合成奶酪魔杖，用于设置老鼠的存放标记",
+  "advancement.rats.cheese_stick.title": "奶酪之触",
+  "advancement.rats.contaminated_food.desc": "发现老鼠留下的恶心的受污染食物",
+  "advancement.rats.contaminated_food.title": "卫生监督快干活！",
+  "advancement.rats.defeat_black_death.desc": "击败黑死神",
+  "advancement.rats.defeat_black_death.title": "不要温和地走进那个良夜",
+  "advancement.rats.garbage_pile.desc": "把方块丢进垃圾桶里获取垃圾堆。可以用来做老鼠刷怪场！",
+  "advancement.rats.garbage_pile.title": "当下的垃圾太多？宇宙中没有垃圾！",
+  "advancement.rats.milk_cauldron.desc": "将牛奶倒入炼药锅中，等待其变成奶酪",
```

```suggestion
  "advancement.rats.milk_cauldron.desc": "将奶倒入炼药锅中，等待其变成奶酪",
```
milk 奶
[Minecraft Wiki:译名标准化/历史](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96/%E5%8E%86%E5%8F%B2)